CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The 'Salary' property should be nullable if it's not required. This ensures compatibility with existing data and avoids potential runtime issues when the value is not set.

```suggestion
public decimal? Salary { get; set; }
```